crates/wasmtime/src/config.rs

```diff
@@ -1837,8 +1831,8 @@ impl Config {
     /// Disabling this will result in a single thread being used to compile
     /// the wasm bytecode.
     ///
-    /// By default parallel compilation is enabled.
-    #[cfg(feature = "parallel-compilation")]
+    /// This feature is enabled by default as long as the `parallel-compilation`
+    /// crate feature (enabled by default) is also enabled.
     pub fn parallel_compilation(&mut self, parallel: bool) -> &mut Self {
         self.parallel_compilation = parallel;
```

Comment:
Mind adding a check below to return an error if this is explicitly enabled but disabled at compile time?